File: app/src/main/java/com/android/keeper/localdb/utilities/RemindersUtilities.java
package com.android.keeper.localdb.utilities;

public class RemindersUtilities {
    public static final String TABLE_NAME="REMINDERS_LIST";

    public static final String COLUMN_REMINDER_ID="reminder_id";
    public static final String COLUMN_REMINDER_TITLE="reminder_title";
    public static final String COLUMN_REMINDER_YEAR="reminder_year";
    public static final String COLUMN_REMINDER_MONTH="reminder_month";
    public static final String COLUMN_REMINDER_DAY="reminder_day";
    public static final String COLUMN_REMINDER_HOUR="reminder_hour";
    public static final String COLUMN_REMINDER_MINUTE="reminder_minute";
    public static final String COLUMN_REMINDER_DONE="reminder_done";

    public static final String CREATE_REMINDER_TABLE="CREATE TABLE "+TABLE_NAME+"("+COLUMN_REMINDER_ID+" INTEGER PRIMARY KEY AUTOINCREMENT NULL,"+
            COLUMN_REMINDER_TITLE+" TEXT NOT NULL,"+COLUMN_REMINDER_YEAR+" INTEGER NULL DEFAULT 0,"+COLUMN_REMINDER_MONTH+" INTEGER NULL DEFAULT 0,"+
            COLUMN_REMINDER_DAY+" INTEGER NULL DEFAULT 0,"+COLUMN_REMINDER_HOUR+" INTEGER NULL DEFAULT 0,"+COLUMN_REMINDER_MINUTE+" INTEGER NULL DEFAULT 0,"+
            COLUMN_REMINDER_DONE+" INTEGER NULL DEFAULT 0)";
}

File: app/src/main/java/com/android/keeper/localdb/utilities/RemindersUtilities.kt
package com.android.keeper.localdb.utilities

import android.support.test.InstrumentationRegistry
import android.support.test.runner.AndroidJUnit4
import org.junit.runner.RunWith

object RemindersUtilities {
    val TABLE_NAME: String? = "REMINDERS_LIST"
    val COLUMN_REMINDER_ID: String? = "reminder_id"
    val COLUMN_REMINDER_TITLE: String? = "reminder_title"
    val COLUMN_REMINDER_YEAR: String? = "reminder_year"
    val COLUMN_REMINDER_MONTH: String? = "reminder_month"
    val COLUMN_REMINDER_DAY: String? = "reminder_day"
    val COLUMN_REMINDER_HOUR: String? = "reminder_hour"
    val COLUMN_REMINDER_MINUTE: String? = "reminder_minute"
    val COLUMN_REMINDER_DONE: String? = "reminder_done"
    val CREATE_REMINDER_TABLE: String? = "CREATE TABLE " + TABLE_NAME + "(" + COLUMN_REMINDER_ID + " INTEGER PRIMARY KEY AUTOINCREMENT NULL," +
            COLUMN_REMINDER_TITLE + " TEXT NOT NULL," + COLUMN_REMINDER_YEAR + " INTEGER NULL," + COLUMN_REMINDER_MONTH + " INTEGER NULL," +
            COLUMN_REMINDER_DAY + " INTEGER NULL," + COLUMN_REMINDER_HOUR + " INTEGER NULL," + COLUMN_REMINDER_MINUTE + " INTEGER NULL," +
            COLUMN_REMINDER_DONE + " INTEGER NULL DEFAULT 0)"
}